code/game/objects/items/stacks/rods.dm

```diff
@@ -92,3 +92,20 @@ GLOBAL_LIST_INIT(rod_recipes, list (
 		user.unEquip(src, 1)
 		if(new_item)
 			user.put_in_hands(new_item)
+
+/obj/item/stack/rods/lava
+	name = "heat resistant rod"
+	desc = "Treated, specialized iron rods. When exposed to the vaccum of space their coating breaks off, but they can hold up against the extreme heat of active lava."
```

Comment:
I kinda feel this should be 'but they can hold up against the extreme heats of liquids' since there are liquid plasma lakes now, though thats just a personal thing.